src/main/java/spoon/reflect/code/CtCatch.java

```diff
@@ -36,12 +36,14 @@
 	/**
 	 * Gets the catch's body.
 	 */
+	@Override
 	CtBlock<?> getBody();
 
 	/**
-	 * Sets the catch's body.
+	 * Sets the catch's body. The instance of CtBlock makes sense too
```

Comment:
we don't need to add it again in CtCatch if we don't change the return type or the contract.